python/grass/__init__.py

```diff
@@ -75,16 +93,48 @@ def _translate(text):
             # does not raise an exception even if the locale settings is broken
             # or the translation files were not installed.
             fallback = True
-            translation = gettext.translation(
-                "grasslibs", locale_dir, fallback=fallback
-            )
-            # Add other domains as fallback.
-            translation.add_fallback(
-                gettext.translation("grassmods", locale_dir, fallback=fallback)
-            )
-            translation.add_fallback(
-                gettext.translation("grasswxpy", locale_dir, fallback=fallback)
-            )
+            translation = None
+            domains = ("grasslibs", "grassmods", "grasswxpy")
+
+            def apply_trans(kwargs):
```

Comment:
You can pass the kwargs parameter really as `**kwargs` which will simplify the _apply_trans_ function calls.